source/connect.txt

```diff
@@ -39,38 +39,74 @@ This is equivalent to the following command:
 Local MongoDB Instance on a Non-Default Port
 --------------------------------------------
 
-To specify a port to connect to on localhost, use the 
-:manual:`connection string </reference/connection-string>` format. 
+To specify a port to connect to on localhost, you can use:
 
-.. example::
+- A :manual:`connection string </reference/connection-string>`. 
 
-   To connect to a MongoDB instance running on localhost with a 
-   non-default port 28015:
+  .. example::
+  
+     To connect to a MongoDB instance running on localhost with a 
+     non-default port 28015:
+  
+     .. code-block:: sh
+  
+        mongosh "localhost:28015"
```

Comment:
```suggestion
        mongosh "mongodb://localhost:28015"
```

Reply:
the commands do the same thing but it's nice to show the protocol here for completeness.

Reply:
consider making this change in the `Local MongoDB Instance on Default Port` section, too.